capsules/src/process_console.rs

```diff
@@ -75,6 +75,65 @@ impl Default for WriterState {
     }
 }
 
+/// Key that can be part from an escape sequence.
+#[derive(Copy, Clone)]
+enum EscKey {
+    Up,
+    Down,
+    Left,
+    Right,
+}
+
+/// Escape state machine to check if
+/// an escape sequence has occured
+#[derive(Copy, Clone)]
+enum EscState {
```

Comment:
Please add a description for all the items.

Reply:
Done.